ext/nnstreamer/extra/nnstreamer_protobuf/meson.build

```diff
@@ -0,0 +1,22 @@
+if protobuf_support_is_available
```

Comment:
Please locate protobuf-related files at ```ext/nnstreamer/extra/```. You don't need to make it deeper until we have a lot more files in extra directory. We may put common files of both protobuf and flatbuf there.

Reply:
Agree. Updated! Thank you.